QualityControl/lib/controllers/LayoutController.js

```diff
@@ -126,47 +124,35 @@ export class LayoutController {
   async putLayoutHandler(req, res) {
     const { id } = req.params;
     try {
-      if (!id) {
-        updateAndSendExpressResponseFromNativeError(res, new InvalidInputError('Missing parameter "id" of layout'));
-      } else if (!req.body) {
-        updateAndSendExpressResponseFromNativeError(
-          res,
-          new InvalidInputError('Missing body content to update layout with'),
-        );
-      } else {
-        const { personid } = req.session;
-        const { owner_id } = await this._dataService.readLayout(id);
-
-        if (Number(owner_id) !== Number(personid)) {
+      let layoutProposed = {};
+      try {
+        if (!req.body) {
           updateAndSendExpressResponseFromNativeError(
             res,
-            new UnauthorizedAccessError('Only the owner of the layout can update it'),
+            new InvalidInputError('Missing request body to update layout'),
           );
-        } else {
-          let layoutProposed = {};
-          try {
-            layoutProposed = await LayoutDto.validateAsync(req.body);
-          } catch (error) {
-            updateAndSendExpressResponseFromNativeError(
-              res,
-              new Error(`Failed to update layout ${error?.details?.[0]?.message || ''}`),
-            );
-            return;
-          }
-
-          const layouts = await this._dataService.listLayouts({ name: layoutProposed.name });
-          const layoutExistsWithName = layouts.every((layout) => layout.id !== layoutProposed.id);
-          if (layouts.length > 0 && layoutExistsWithName) {
-            updateAndSendExpressResponseFromNativeError(
-              res,
-              new InvalidInputError(`Proposed layout name: ${layoutProposed.name} already exists`),
-            );
-            return;
-          }
-          const layout = await this._dataService.updateLayout(id, layoutProposed);
-          res.status(201).json({ id: layout });
+          return;
         }
+        layoutProposed = await LayoutDto.validateAsync(req.body);
+      } catch (error) {
+        updateAndSendExpressResponseFromNativeError(
+          res,
+          new InvalidInputError(`Failed to update layout ${error?.details?.[0]?.message || ''}`),
+        );
+        return;
       }
+
+      const layouts = await this._dataService.listLayouts({ name: layoutProposed.name });
+      const layoutExistsWithName = layouts.every((layout) => layout.id !== layoutProposed.id);
+      if (layouts.length > 0 && layoutExistsWithName) {
+        updateAndSendExpressResponseFromNativeError(
+          res,
+          new InvalidInputError(`Proposed layout name: ${layoutProposed.name} already exists`),
+        );
+        return;
+      }
+      const layout = await this._dataService.updateLayout(id, layoutProposed);
```

Comment:
This method will return a `layoutId`. Could you please update the name of the variable accordingly?
```suggestion
      const layoutId = await this._dataService.updateLayout(id, layoutProposed);
```

Reply:
Done